kernel/main.c

```diff
@@ -340,6 +353,52 @@ async_init(void)
       panic("Unable to create a kthread for do_async_init()");
 }
 
+static void
+setup_uefi_runtime_services() {
```

Comment:
Tilck should be portable as much as possible. Probably, this function should be moved to a dedicated file and be renamed something a bit more generic as "setup_firmware_runtime_services". But, since nothing is working at the moment, that's fine. Let's keep it as a note for later.